### PR TITLE
Add support for docExpansion option

### PR DESCRIPTION
**Pull request by maruware**
This Pull Request will support to use docExpansion option.

```ruby
GrapeSwaggerRails.options.doc_expansion = 'list'
```

### CODE REVIEW

**Comment by dblock** [comment]
This needs a CHANGELOG entry, README updates and some tests to be merged, please.

**Comment by maruware** [comment]
CHANGELOG, README, tests updated.
Check those, please.

**Comment by dblock** [comment]
Squash commits please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * Upgraded swagger-ui to v2.1.1 - [@dblock](https://github.com/dblock).
 * Grape-swagger 0.7.2 is no longer supported - [@dblock](https://github.com/dblock).
 * Implemented RuboCop, Ruby-style linter - [@dblock](https://github.com/dblock).
+* Support Swagger-UI docExpansion option - [@maruware](https://github.com/maruware).
```
Comment (dblock) [inline CHANGELOG.md]: See examples below, this should have the PR and your name.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,12 @@ You can specify additional headers to add to each request:
 GrapeSwaggerRails.options.headers['Special-Header'] = 'Some Secret Value'
 ```
 
+You can set docExpansion with "none" or "list" or "full", default is "none"
```
Comment (dblock) [inline README.md]: Explain what that actually does?

**Comment by dblock** [comment]
Minor comments above. Otherwise looks great.

**Comment by maruware** [comment]
Thank you for the advice. Those points updated.

**Comment by dblock** [comment]
Perfect, thanks.

**Pull request merged**